docs/index.html

```diff
@@ -81,9 +81,9 @@ <h3>Chart Type</h3>
   <!-- optioinally load the amcharts plugin to export the chart as and image or table -->
   <script src="https://www.amcharts.com/lib/3/plugins/export/export.min.js"></script>
   <link rel="stylesheet" href="https://www.amcharts.com/lib/3/plugins/export/export.css" type="text/css" media="all" />
-  <!-- load the arcgis-rest-js scripts -->
-  <script src="https://unpkg.com/@esri/arcgis-rest-request@1.7.1/dist/umd/request.umd.min.js"></script>
-  <script src="https://unpkg.com/@esri/arcgis-rest-feature-service@1.7.1/dist/umd/feature-service.umd.min.js"></script>
+  <!-- you should have to load arcgis-rest-js scripts, but right now cedar bundles them! -->
```

Comment:
This should have been taken care of in https://github.com/Esri/cedar/pull/421/files - I must have missed something, or done something wrong...

Reply:
i'll tackle this in #391